pkg/controller/installation/installation_controller.go

```diff
@@ -328,6 +361,18 @@ func (r *ReconcileInstallation) preflightChecks(installation *integreatlyv1alpha
 		}
 	}
 
+	if integreatlyv1alpha1.InstallationType(installation.Spec.Type) == integreatlyv1alpha1.InstallationTypeWorkshop {
+		installation.Status.PreflightMessage = "preflight checks passed"
+		installation.Status.PreflightStatus = integreatlyv1alpha1.PreflightSuccess
+		err := r.client.Status().Update(r.context, installation)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
+		return reconcile.Result{
```

Comment:
Is the `retryRequeue` variable in scope here? If so, it looks like it might be useful here :slightly_smiling_face:

Reply:
it is not, I'm not convinced it's worth exposing it for this one use either. WDYT?